File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int i = 0; i < nums.length; i++){
            if(digits(nums[i]) % 2 == 0){
                count++;
            }
        }
        return count;
    }
    
    public int digits(int num){
        int no = 0;
        while(num > 0){
            no++;
            num /= 10;
        }
        return no;
    }
} 

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        count = 0
        for i in nums:
            if self.digits(i) % 2 == 0 :
                count += 1
        return count
    
    def digits(self, num):
        no = 0
        while num > 0:
            no += 1
            num = num // 10
        return no